updated winui3/reunion doc (#1286)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/get-started/winui.md

ms.technology: webview
keywords: WebView2, webview2, WebView, webview, winui apps, winui, edge, CoreWebView2, browser control, edge html, get started, Get Started, .NET
---
# Get started with WebView2 in WinUI 3 - WindowsAppSDK

In this article, get started creating your first WebView2 app and learn about the main features of [WebView2][MicrosoftDeveloperMicrosoftEdgeWebview2].  Your first WebView2 app uses WinUI3.  For more information on individual APIs, navigate to [API reference][GithubMicrosoftMicrosoftUiXamlSpecsWebview2].  
        
## Step 0 - Set Up Development Environment 

> 1. Follow steps 1-4 of the [WindowsAppSDK environment setup guide][WindowsAppsWinui3ConfigureYourDevEnvironment]. This guide will walk you through minimum system requirements, installing visual studio, enabling NuGet package source, and installing the WindowsAppSDK Extension for Visual Studio. 

Reviewer: ```suggestion
1. Follow steps 1-4 of the [WindowsAppSDK environment setup guide][WindowsAppsWinui3ConfigureYourDevEnvironment]. This guide describes the minimum system requirements, installing visual studio, enabling NuGet package source, and installing the WindowsAppSDK Extension for Visual Studio. 
```

1. Install [WebView2 Runtime][Webview2Installer] or any [Microsoft Edge (Chromium) non-stable channel][MicrosoftedgeinsiderDownload] installed on Windows 10 version 1803 \(build 17134\) or later.  For more information about Windows 10, navigate to [Windows Update: FAQ][MicrosoftSupport12373].  
1.  To access all developer-specific Visual Studio features, turn on [Developer Mode][WindowsUwpGetStartedEnableYourDeviceForDevelopment].  
    

> ## Step 1 - Create Project  

Reviewer: Step 2 says "choose C#, Windows, and WinUI respectively" but the image (winui-getting-started-selections.png) seems to show C#, Windows, Desktop, and WinUI. Do the text or image need to be changed to match each other?

Author: text can be changed to specify desktop

Author: Actually the drop down selections only have spots for C#, Windows, WinUI, there is no place to choose Desktop, In the highlighted selection that is just an extra tag. We can leave it as is for now.

> As an example, add scripts that send an alert when a user navigates to non-HTTPS sites.  Modify the `EnsureHttps` function to inject a script into the web content that uses [ExecuteScriptAsync][Webviews2ReferenceWpfMicrosoftWebExecutescriptasync].  

Reviewer: Also, in the second bullet under the "Scripting" heading, the second bullet: "Run it before any other script included in the HTML document is run." reads better as "Run it before any other script included in the HTML document runs."

Author: changed
